response_operations_ui/controllers/reporting_units_controllers.py

```diff
@@ -67,6 +67,10 @@ def change_respondent_status(respondent_id, change_flag):
             raise ApiError(response)
 
         logger.debug('Successfully changed respondent status', respondent_id=respondent_id, change_flag=change_flag)
+    else:
+        error = 'Incorrect change_flag given'
+        logger.error(error)
```

Comment:
You shouldn't have to make a string of a message and pass it through. Can do what happens on line 66 such as 
```python
logger.error('Incorrect change_flag given', respondent_id=respondent_id, change_flag=change_flag)
```

Reply:
Agreed, but the next line is raising a ValueError with the same text.  Might as well not repeat ourselves

Reply:
changed